caluma/caluma_form/models.py

```diff
@@ -291,7 +294,26 @@ class Meta:
         indexes = [GinIndex(fields=["meta"])]
 
 
-class Answer(UUIDModel):
+class QuestionTypeHistoricalModel(models.Model):
+    """Extra fields for HistoricalAnswer."""
+
+    history_question_type = models.CharField(
+        choices=Question.TYPE_CHOICES_TUPLE, max_length=23
+    )
+
+    class Meta:
+        abstract = True
+
+
+class Answer(core_models.BaseModel):
+    """Records an answer to a question of arbitrary type.
+
+    We need to replicate the UUIDModel in order to register it as historical model.
```

Comment:
I think this is better put in a comment rather than the docstring. No need to put implementation detail into "public" docstrings.

Reply:
Done